ibc-core/ics24-host/cosmos/src/upgrade_proposal/mod.rs

```diff
@@ -8,6 +8,6 @@ mod proposal;
 
 pub use context::*;
 pub use events::{UpgradeChain, UpgradeClientProposal};
-pub use handler::upgrade_client_proposal_handler;
+pub use handler::execute_upgrade_client_proposal;
 pub use plan::Plan;
-pub use proposal::UpgradeProposal;
+pub use proposal::*;
```

Comment:
This exports `UPGRADE_PROPOSAL_TYPE_URL`. Is this intended? I see, the other files don't exports the type URLs consts.

Reply:
Yes, this is intended, as that type URL is used in basecoin.